sfxclient/sfxclient.go

```diff
@@ -110,6 +116,10 @@ func (h *HashableCollector) Datapoints() []*datapoint.Datapoint {
 
 var _ Collector = CollectorFunc(nil)
 
+const (
+	spanDatapointSuffix = "-datapoint"
```

Comment:
a little confused as to what "-datapoint" is trying to convey here?

Reply:
it was added to represent what cb function datapoint is getting collected. However, parent trace name being "collect-datapoint" this can be removed